dist/options/options.html

```diff
@@ -223,6 +226,18 @@ <h1>
 
             <textarea id="label-list" rows="16"></textarea>
 
+            <h2>
+                Group by date
+            </h2>
+
+            <div>
+                <label class="switch">
+                    <input type="checkbox" id="group-by-date-checkbox">
+                    <span class="slider"></span>
+                </label>
+                <span class="switch-label">Group messages by date</span>
+            </div>            
+
             <button id="save-button" class="save-button">
```

Comment:
Could you add some vertical whitespace before and after the save button? (It looks a little squished right now.)

Reply:
Done!